### PR TITLE
port from core_optional

### PR DESCRIPTION
**Pull request by WAvdBeek**

### CODE REVIEW

File: swagger2.0/oic.r.mqtt.conf.swagger.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/mqttconfResURI" : {
+      "get": {
+        "description": "The Resource through the MQTT server information can be set.\n",
```
Comment (marktrayer) [inline swagger2.0/oic.r.mqtt.conf.swagger.json]: Small grammar nit, change text to:
The Resource through which the MQTT server information can be set.

File: swagger2.0/oic.r.mqtt.conf.swagger.json [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      "name" : "if",
+      "type" : "string",
+      "enum" : ["oic.if.rw", "oic.if.baseline"]
+    }
+  },
+  "definitions": {
+    "mqttconf" : {
+      "properties": {
+        "server" : {
+          "description": "The connection information of the MQTT server. Can be an URI or IP address",
```
Comment (marktrayer) [inline swagger2.0/oic.r.mqtt.conf.swagger.json]: Small grammar nit:
"an URI" -> "a URI"

File: swagger2.0/oic.r.mqtt.conf.swagger.json (hunk 1 above)
Comment (WAvdBeek) [reply swagger2.0/oic.r.mqtt.conf.swagger.json]: fixed

File: swagger2.0/oic.r.mqtt.conf.swagger.json (hunk 2 above)
Comment (WAvdBeek) [reply swagger2.0/oic.r.mqtt.conf.swagger.json]: fixed

**Pull request merged**